Automattic/woocommerce-payments
# Update links across the plugin to woo.com

Fixes #7637 

#### Changes proposed in this Pull Request





- Update hyperlinks across the plugin to refer to `woo.com` rather than `woocommerce.com`
- Update plugin URL to point to `woo.com` in the `woocommerce-payments.php` file.




#### Testing instructions





* Spot check a few of the URLs in the diff to ensure they go to the correct woo.com documentation page.
* Check a few of the links in the plugin (for example, there are some hyperlinks on the `Payments > Settings` page like the Multi-Currency documentation link).

-------------------


**Post merge**
